2019-05-Berlin/agenda.md

```diff
@@ -81,8 +94,6 @@ Working groups will put together a brief schedule so that people can
 familiarize themselves before folks get onsite, having the general collaborator 
 discussions, and then dive into breakout sessions.
 
-Attendees must register at: (TBA)
```

Comment:
would this be added back again? Or we are not doing any registration for attendees?

Reply:
We hadn't talked about doing it so I took it out.  I also wasn't sure if it referred to onsite registration or the evenbrite.

Reply:
I think we can add [Eventbrite Registration Link]( https://www.eventbrite.com/e/openjs-collaborator-summit-tickets-55985971548) here with the password?

@ladyleet

Reply:
+1 to adding the Eventbrite link somewhere in here.

Reply:
Just added it and the password